flows/definition/legacy/definition_test.go

```diff
@@ -75,21 +74,21 @@ var legacyRuleSetHolderDef = `
 	"action_sets": [
 		{
 			"uuid": "5b977652-91e3-48be-8e86-7c8094b4aa8f",
-			"x": 0, "y": 200, 
+			"x": 0, "y": 2200, 
```

Comment:
bumping these so the migrated node for the ruleset always ends up being the first node in the list